ui/app/components/meta-table/index.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="artifact-overview">
+  <table>
+    <tbody>
+      {{#if @statusReport}}
+        <tr>
+          <td>
```

Comment:
You can use a [row-scoped table header](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/th#attr-scope) here, if you like.

```suggestion
          <th scope="row">
```